Razor/Razor.sublime-syntax

```diff
@@ -0,0 +1,338 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/syntax.html
+name: HTML (C# Razor)
+scope: text.html.cs.razor
+version: 2
+
+extends: Packages/HTML/HTML.sublime-syntax
+
+file_extensions:
+  - cshtml
+
+variables:
+  csharp_tag: '@{'
+  csharp_tag_end: '}'
+  csharp_tag_interpolation_paren: '@\('
+  csharp_tag_interpolation_bare: '@(?=\w)'
+
+contexts:
+  prototype:
+    - meta_prepend: true
+    - include: comments
+    - include: csharp-embedded
+
+  main:
+    - meta_include_prototype: false
+    - match: ''
+      push: [razor, directives]
+
+  razor:
+    # This is the main Razor context.
+    # It includes `csharp-tags` which are allowed to be closed by } tags at any
+    # level of code-block on stack resuming with the next top-level @{ tag.
+    - meta_include_prototype: false
+    - include: csharp-tags
+    - include: html
+
+  comments:
+    - match: '@\*'
+      scope: punctuation.definition.comment.block.begin.cs-razor
+      push:
+        - meta_scope: comment.block.cs-razor
+        - match: '\*@'
+          scope: punctuation.definition.comment.block.end.cs-razor
+          pop: 1
+
+###[ DIRECTIVES ]################################################################
+
+  directives:
+    - meta_include_prototype: false
+    - include: comments
+    - match: <!--
+      push:
+        - meta_scope: region.pinkish
```

Comment:
lol this surprised me when I opened a cshtml file - I was like, why are these comments highlighted pink :sweat_smile: